cluster-autoscaler/clusterstate/clusterstate_test.go

```diff
@@ -722,3 +722,35 @@ func TestGetClusterSize(t *testing.T) {
 	assert.Equal(t, 3, currentSize)
 	assert.Equal(t, 30, targetSize)
 }
+
+func TestIsNodeNotStarted(t *testing.T) {
```

Comment:
rename the the test method to match renamed name :)

Reply:
Done, thanks for catching this